src/object-number.scss

```diff
@@ -0,0 +1,71 @@
+@import "./new-settings";
+@import "./mixins";
+
+/*!
+.fd-object-number+(--success | --warning | --error)
+*/
+$block: #{$fd-namespace}-object-number;
+
+$color-states: (
+  "positive": ("color": var(--sapPositiveTextColor)),
+  "critical": ("color": var(--sapCriticalTextColor)),
+  "negative": ("color": var(--sapNegativeTextColor)),
+  "informative": ("color": var(--sapInformativeTextColor))
+);
+
+.#{$block} {
+  // TEXT VARIABLES
+  $fd-object-number-font-size: var(--sapFontSize) !default;
+  $fd-object-number-neutral: var(--sapNeutralTextColor);
```

Comment:
this doesn't make sense to be a variable since it's used in only one place in the code.